packages/web/src/common/store/social/tracks/recordListen.ts

```diff
@@ -19,11 +19,15 @@ function* recordListen(action: { trackId: number }) {
   const { trackId } = action
   const audiusBackendInstance = yield* getContext('audiusBackendInstance')
 
-  yield* waitForWrite()
+  yield* call(waitForWrite)
   const userId = yield* select(getUserId)
   const track = yield* select(getTrack, { id: trackId })
   if (!userId || !track) return
 
+  if (userId === track.owner_id && (track.listenCount ?? 0) > 10) {
```

Comment:
I remember this logic but was it dropped at some point??

Reply:
cc: @raymondjacobson do we want this or is it all handled in ACDC/indexing anyway?

Reply:
none of this touches ACDC, and it's not handled in indexing. this is a good change to keep, but I think it's fine either way. It was more relevant for when the entire network had very low traffic on trending

Reply:
duh, right we just talked about this 🤦 thanks